### ISSUE TITLE
[False Positive] JsonEncodingApiUsageInspection

### ISSUE THREAD

**Issue by puggan** [kalessil/phpinspectionsea#1483]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Ultimate) v3.0.6 |
| Language level | PHP 7.4 |

Current behaviour
--
```php
    echo json_encode([], JSON_THROW_ON_ERROR);
    $options = JSON_THROW_ON_ERROR;
    echo json_encode([], $options);
```
![Screenshot from 2020-01-09 15-12-04](https://user-images.githubusercontent.com/866668/72074798-db3e7780-32f2-11ea-81f5-025c4929eca5.png)

Expected behaviour
--
no JsonEncodingApiUsageInspection if the option-parameter is a variable that may include JSON_THROW_ON_ERROR

Environment details
---
PhpStorm 2019.3.1
Build #PS-193.5662.63, built on December 19, 2019
Runtime version: 11.0.5+10-b520.17 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 5.3.0-26-generic
GC: ParNew, ConcurrentMarkSweep
Memory: 1923M
Cores: 8
Registry: run.processes.with.pty=TRUE, documentation.show.toolbar=true
Non-Bundled Plugins: org.jetbrains.plugins.rest, Shell Process, String Manipulation, SvgViewer 2, com.aurimasniekis.phppsr4namespacedetector, com.kalessil.phpStorm.phpInspectionsUltimate, com.raket.silverstripe, com.wix.eslint, de.espend.idea.laravel, in.xiv.cs, intellij.prettierJS, izhangzhihao.rainbow.brackets, mobi.hsz.idea.gitignore, name.kropp.intellij.makefile, net.king2500.plugins.PhpAdvancedAutoComplete, net.seesharpsoft.intellij.plugins.csv, net.vektah.codeglance, org.intellij.RegexpTester, org.psliwa.idea.composer

**Comment by kalessil** [comment]
Oh, I love edge-cases.

**Comment by kalessil** [comment]
Fixed!
